.expeditor/verify.pipeline.yml

```diff
@@ -2,29 +2,27 @@ steps:
 
 - label: lint-chefstyle
   command:
-    - bundle install --jobs=7 --retry=3 --without docs debug
-    - bundle exec rake style
+    - .expeditor/run_linux_tests.sh "rake style"
```

Comment:
Looks like we're losing `--without docs debug`. That was originally specified in order to limit the number of gems downloaded, correct?

Reply:
Yep but now we cache it all so there’s no reason to limit it